### PR TITLE
doc/jaegertracing: refine index.rst

### PR DESCRIPTION
**Pull request by zdover23**
Make English in jaegertracing/index.rst more elegant.

Signed-off-by: Zac Dover <zac.dover@gmail.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/jaegertracing/index.rst [hunk 1]
```diff
@@ -1,91 +1,98 @@
...
-there is also `Upload id` tag for Multipart upload operations.
+There is also an `Upload id` tag for Multipart upload operations.
 
-request trace is named `<command> <transaction id>`.
+The names of request traces have the following format: `<command> <transaction
+id>`.
 
 MULTIPART UPLOAD
 ^^^^^^^^^^^^^^^^
-there is a trace, that consists a span for each request that made by that multipart upload, including all `Put Object` requests.
```
Comment (zdover23) [inline doc/jaegertracing/index.rst:86]: @neha-ojha What's a multipart upload?

**Review by anthonyeleven — APPROVED** [review]

File: doc/jaegertracing/index.rst (hunk 1 above)
Comment (anthonyeleven) [reply doc/jaegertracing/index.rst:86]: Big RGW objects can be uploaded in peices

Comment (zdover23) [reply doc/jaegertracing/index.rst:86]: @anthonyeleven Thanks, Anthony. Good looking out.

**Comment by zdover23** [comment]
New in Reef -- no backports yet.

File: doc/jaegertracing/index.rst (hunk 1 above)
Comment (anthonyeleven) [reply doc/jaegertracing/index.rst:86]: Given the context here I don't think explaining the term is warranted.

**Pull request merged**